lib/config.ts

```diff
@@ -140,6 +149,7 @@ export class Config {
     this.loglevel = options.loglevel ?? Config.LOGLEVEL_DEFAULT
     this.minPeers = options.minPeers ?? Config.MINPEERS_DEFAULT
     this.maxPeers = options.maxPeers ?? Config.MAXPEERS_DEFAULT
+    this.chain = options.chain
```

Comment:
Chain should then also be completely initialized here, either with the option or separately so that from this moment on it is assured that a chain exists.